.github/workflows/browser-Extension-Release-Chromwe-Webstore.yml

```diff
@@ -0,0 +1,35 @@
+name: "Browser-Extension-Release-Chrome-Webstore"
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'browser-extension/**'
+  release:
+    types: [created] 
+
+jobs:
+  chrome-extension:
+    name: "zip-files"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: papeloto/action-zip@v1
+        with:
+          files: browser-extension/
+          dest: starfix-extension-chrome.zip
+
+      - name: "publish extension on chrome webstore"
+        uses: Klemensas/chrome-extension-upload-action@v1.3
+        with:
+          refresh-token: 'xyz'
+          client-id: 'xyz.apps.googleusercontent.com'
+          file-name: ${{ github.workspace }}/starfix-extension-chrome.zip
```

Comment:
should this not just be "starfix-extension-chrome.zip" ?

Reply:
Yes

Reply:
_Here are the logs after making suggested changes._

Run Klemensas/chrome-extension-upload-action@master
/usr/bin/docker run --name e1cc51461cdf9c760342b19e7985ff94784103_da7574 --label e1cc51 --workdir /github/workspace --rm -e INPUT_REFRESH-TOKEN -e INPUT_CLIENT-ID -e INPUT_FILE-NAME -e INPUT_APP-ID -e INPUT_PUBLISH -e INPUT_CLIENT-SECRET -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/browser-extension-for-starfix/browser-extension-for-starfix":"/github/workspace" e1cc51:461cdf9c760342b19e7985ff94784103  "***" "***" "" "starfix-extension-chrome.zip" "ojlhpgekmmemhihkigibkhehlejkjdmo" "true"
*   Trying 172.217.9.202...
* TCP_NODELAY set
* Connected to www.googleapis.com (172.217.9.202) port 443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [102 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [4392 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [115 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [37 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-ECDSA-CHACHA20-POLY1305
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: CN=upload.video.google.com
*  start date: Sep 13 03:22:41 2021 GMT
*  expire date: Nov 20 03:22:40 2021 GMT
*  subjectAltName: host "www.googleapis.com" matched cert's "*.googleapis.com"
*  issuer: C=US; O=Google Trust Services LLC; CN=GTS CA 1C3
*  SSL certificate verify ok.
> POST /oauth2/v4/token HTTP/1.1
> Host: www.googleapis.com
> User-Agent: curl/7.54.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 271
> 
} [271 bytes data]
* upload completely sent off: 271 out of 271 bytes
< HTTP/1.1 200 OK
< Cache-Control: no-cache, no-store, max-age=0, must-revalidate
< Date: Thu, 30 Sep 2021 08:40:52 GMT
< Pragma: no-cache
< Expires: Mon, 01 Jan 1990 00:00:00 GMT
< Content-Type: application/json; charset=utf-8
< Vary: X-Origin
< Vary: Referer
< Server: scaffolding on HTTPServer2
< X-XSS-Protection: 0
< X-Frame-Options: SAMEORIGIN
< X-Content-Type-Options: nosniff
< Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000,h3-T051=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
< Accept-Ranges: none
< Vary: Origin,Accept-Encoding
< Transfer-Encoding: chunked
< 
{ [303 bytes data]
* Connection #0 to host www.googleapis.com left intact
*   Trying 172.217.12.234...
* TCP_NODELAY set
* Connected to www.googleapis.com (172.217.12.234) port 443 (#0)
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
} [512 bytes data]
* TLSv1.2 (IN), TLS handshake, Server hello (2):
{ [102 bytes data]
* TLSv1.2 (IN), TLS handshake, Certificate (11):
{ [4392 bytes data]
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
{ [115 bytes data]
* TLSv1.2 (IN), TLS handshake, Server finished (14):
{ [4 bytes data]
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
} [37 bytes data]
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
} [1 bytes data]
* TLSv1.2 (OUT), TLS handshake, Finished (20):
} [16 bytes data]
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
{ [1 bytes data]
* TLSv1.2 (IN), TLS handshake, Finished (20):
{ [16 bytes data]
* SSL connection using TLSv1.2 / ECDHE-ECDSA-CHACHA20-POLY1305
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: CN=upload.video.google.com
*  start date: Sep 13 03:22:41 2021 GMT
*  expire date: Nov 20 03:22:40 2021 GMT
*  subjectAltName: host "www.googleapis.com" matched cert's "*.googleapis.com"
*  issuer: C=US; O=Google Trust Services LLC; CN=GTS CA 1C3
*  SSL certificate verify ok.
> PUT /upload/chromewebstore/v1.1/items/ojlhpgekmmemhihkigibkhehlejkjdmo HTTP/1.1
> Host: www.googleapis.com
> User-Agent: curl/7.54.0
> Accept: */*
> Authorization: ***
> x-goog-api-version: 2
> Content-Length: 2477
> Expect: 100-continue
> 
< HTTP/1.1 100 Continue
} [2477 bytes data]
* We are completely uploaded and fine
< HTTP/1.1 200 OK
< X-GUploader-UploadID: ADPycdufL8oOZd-gIP9njsbHrmh7Yo4KsHKtWEthxr8xFXIWlFh0KobI3CjKFWf4UdRugsazroL6GYWG7S2GSL0GZGjyVkThww
< Content-Type: application/json; charset=UTF-8
< Vary: Origin
< Vary: X-Origin
< Vary: Referer
< Content-Length: 356
< Date: Thu, 30 Sep 2021 08:40:53 GMT
< Server: UploadServer
< Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000,h3-T051=":443"; ma=2592000,h3-Q050=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000,quic=":443"; ma=2592000; v="46,43"
< 
{ [356 bytes data]
* Connection #0 to host www.googleapis.com left intact

Reply:
@maxandersen I'm a bit stuck up on this. I've tried a bunch of things but I am unable to figure out why it's not working.
You can refer my repository for logs: https://github.com/fahad-israr/browser-extension-for-starfix/actions
(There are too many of them because I've tried a lot of times )